guide/07-working-with-bigdata/10-extending_geoanalytics_with_custom_tools.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p>In this example, we will find hot spots of delayed buses. To do this, a big data file share dataset has been specified in the <code>input__layers</code> parameter of the Run Python Script task. It is available within the script at index <code>0</code> of the layers object. This dataset contains point locations of city busses recorded at 1 minute intervals for 15 days. Each point feature contains a status field called <code>dly</code> that is <code>True</code> if the bus is delayed and <code>False</code> in all other instances.</p><p> </p><p><strong>Chaining together GeoAnalytics Tools to find hot spots of bus delay points.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Wording changes in the first paragraph.</p>